packages/components/organisms/f-cookie-banner/package.json

```diff
@@ -32,7 +32,7 @@
   "scripts": {
     "prepublishOnly": "yarn lint && yarn test && yarn build && yarn build:static-files",
     "build": "vue-cli-service build --target lib --name f-cookie-banner ./src/index.js",
-    "build:static-files": "yarn --cwd ./f-cookie-banner-static build",
+    "build:static-files": "yarn --cwd ../f-cookie-banner-static build",
```

Comment:
Seems weird to have the build for the static banner in this package.json now – could this not be moved to the `f-cookie-banner-static` package.json?

Reply:
Nope. This script is what gets executed by `prepublishOnly`.

This ensures that when we change is made to `f-cookie-banner`, the static files get bundled as part of the package release.